development/CohortOntology.owl

```diff
@@ -1230,14 +1127,7 @@ NOTE: study design options are mutually exclusive for Cohort Groups but not Proj
     <!-- https://w3id.org/enigma#usesScanner -->
 
     <owl:DatatypeProperty rdf:about="https://w3id.org/enigma#usesScanner">
-        <rdfs:domain>
-            <owl:Class>
-                <owl:unionOf rdf:parseType="Collection">
-                    <rdf:Description rdf:about="https://w3id.org/enigma#Cohort"/>
-                    <rdf:Description rdf:about="https://w3id.org/enigma#CohortGroup"/>
-                </owl:unionOf>
-            </owl:Class>
-        </rdfs:domain>
+        <rdfs:domain rdf:resource="https://w3id.org/enigma#Cohort"/>
```

Comment:
Why here just cohort; and in other you add the domain to be cohort group and project? They are going to inherit this property too.

Reply:
I modified this property after you told me that including just Cohort is enough - so I did not change the domain.

Reply:
Ok, what I think it's a little inconsistent is having the 3 in some cases, and just having cohort in some cases. I would keep it consistent overall, adding here the other 2.